Assets/MRTK/SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -82,15 +82,16 @@ public virtual void SetCursor(GameObject newCursor = null)
         {
             using (SetCursorPerfMarker.Auto())
             {
-                if (cursorInstance != null)
+                // Destory the old cursor and replace it with the new one if a new cursor was provided
+                if (cursorInstance != null && newCursor != null)
                 {
                     DestroyCursorInstance();
                     cursorInstance = newCursor;
                 }
 
                 if (cursorInstance == null && cursorPrefab != null)
                 {
-                    cursorInstance = Instantiate(cursorPrefab, transform);
+                    cursorInstance = Instantiate(cursorPrefab, transform.parent);
```

Comment:
There _might_ be some assumptions in the toolkit about the hierarchy here, but none specifically come to mind and hopefully our tests cover it!

Reply:
I'll add a comment documenting the behavior for now. Hopefully we don't need this code when we rework it :)